app/main.py

```diff
@@ -1,8 +1,63 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(
+        self,
+        comfort_class: int,
+        clean_mark: int,
+        brand: str
+    ) -> None:
+        if not (1 <= comfort_class <= 7):
+            print("Error")
+        if not (1 <= clean_mark <= 10):
+            print("Error")
```

Comment:
What if I add this code at the end?
```
wrong_car = Car(-100, -100, "name")
Car.serve_cars([wrong_car])
```
It'll print this text, but the program will continue to work with incorrect data, thus you should either set correct data or stop program